2.0/runtime/nanoserver-insider/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# escape=`
+
+# Installer image
+FROM microsoft/windowsservercore-insider AS installer-env
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+ENV DOTNET_VERSION 2.0.0-preview3-25510-01
```

Comment:
The SDK image has a comment `# Install .NET Core SDK`, maybe add `# Install .NET Core Runtime` here?